docs/getting-started/quickstart.md

```diff
@@ -93,157 +117,89 @@ crate-type = ["cdylib"]
 
 [dependencies]
 substreams = "0.5.0"
+substreams-ethereum = "0.8.0"
 prost = "0.11"
+num-bigint = "0.4"
```

Comment:
I don't think this is one is strictly required for the tutorial

Reply:
Removed.

Reply:
Can you test it though just to be sure it still works without.

Reply:
Yes. Tested when I updated the code example. Appears to be fully operational.